internal/cli/opsmanager/metrics/databases_list.go

```diff
@@ -38,6 +38,10 @@ func (opts *DatabasesListsOpts) initStore() error {
 	return err
 }
 
+var DatabasesListTemplate = `NAME{{range .Results}}
```

Comment:
I'd say we can do without the title here, rule of thump here, if we have a single column and is easy to understand what we are returning we can do without a title

Reply:
Also let's not export this, `databasesListTemplate`